index.html

```diff
@@ -48,6 +49,172 @@ <h4 class="social-title">LET'S CONNECT</h4>
+
+                <div class="card-content4">
+                    <ul class="type4">
+                        <p class="t1">CANOPY</p>
+                        <p class="t2"><img src="./images/Circle.png"/> Back End Dev</p>
+                        <p class="t3"><img src="./images/Circle.png"/> 2015</p>
+                    </ul>
+
+                    <p class="text4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+                    <ul class="tech4">
+                        <li class="language1">html</li>
+                        <li class="language2"> css</li>
+                        <li class="language3">javascript</li>
+                    </ul>
+
+                    <button type="button" class="see-proj4">See Project</button>
+                </div>
+            </li>
+        </ul>
+    </section>
+                                    <!-- Section 2 : About Myself -->
+    <section class="about-myself">
+        <div class="about-container">
+            <h3 class="about-myself-title">About Myself</h3>
+            <p class="about-myself-resume">
+                Hello I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hesitate to contact me.
+            </p>
+            <h4 class="connect">LET'S CONNECT</h4>
+            <ul class="social-media">
+                <li>
+                  <a href="#"><img src="./images/social/Github.svg" alt="Github" /></a>
+                </li>
+                
+                <li>
+                  <a href="#"><img src="./images/social/Linkedin.svg" alt="Linkedln"/></a>
+                </li>
+                
+                <li><a href="#"><img src="./images/social/Hand2.svg" alt="" /></a></li>
+                
+                <li><a href="#"><img src="./images/social/Twitter.svg" alt="Twitter"/></a></li>
+                
+                <li><a href="#"><img src="./images/social/Vector-M.svg" alt="medium-icon" /></a></li>
+
+            </ul>
+            
+            <button type="button" onclick="" class="see-project">
+                Get my resume
+            </button>
+        </div>
+        <ul class="my-resume">
+            <li class="resume-title">Languages<img src="./images/arrow-icon.png" alt="Arrow Down" class="arrow1"/>
+            </li>
+
+            <li>
+                <ul class="prog-languages">
+                    <p class="javascript"><img src="./images/js.png" alt="JavaScript"/></p>
+                    <h4>JavaScript</h4>
+                    <p class="html"><img src="./images/html.png" alt="HTML"/></p>
+                    <h4>HTML</h4>
+                    <p class="css"><img src="./images/ic-css.png" alt="CSS"/></p>
+                    <h4>CSS</h4>    
+                </ul>
+
+            </li>
```

Comment:
I believe you might have mistakenly put `ul` tag inside a `li` tag. But it is actually the opposite. The `li` tag comes between the `ul` or `ol` tags, not the otherwise.